jax/_src/scipy/special.py

```diff
@@ -956,3 +956,56 @@ def lpmn(m, n, z):
   p_derivatives = _gen_derivatives(p_vals, z, is_normalized)
 
   return (p_vals, p_derivatives)
+
+
+def lpmn_values(m, n, z, is_normalized):
+  """The associated Legendre functions (ALFs) of the first kind.
+
+  Unlike `lpmn`, this function only computes the values of ALFs.
+  The ALFs of the first kind can be used in spherical harmonics. The
+  spherical harmonic of degree `l` and order `m` can be written as
+  `Y_l^m(θ, φ) = N_l^m * P_l^m(cos(θ)) * exp(i m φ)`, where `N_l^m` is the
```

Comment:
You could use a `:math:` directive here to make this expression render more cleanly in the HTML docs.

Reply:
Done. Thanks!